app/views/trainees/_action_bar.html.erb

```diff
@@ -1,12 +1,16 @@
 <% unless paginated_trainees.empty? %>
   <%= render SortLinks::View.new %>
   <% if FeatureService.enabled?(:trainee_export) && policy(:trainee).export? %>
```

Comment:
`policy(:trainee).export?` is what controls the permissions. I was wondering about editing the trainee_policy to give it access to the number of filtered trainees, so we could restrict at this point, but it didn't feel right as that file is more focused around user permissions. I might be wrong though and maybe that would be ok.